### ISSUE TITLE
Distribution via package managers

### ISSUE THREAD

**Issue by EivindArvesen** [eivind88/prm#5]
prm should be distributed through various package managers (e.g. apt, brew, macports, pacman, yum...), though it should perhaps be postponed until there is a stable release.

Needs to include an append (alias) to a user dotfile or at least a message regarding proper use of prm (i.e. sourcing the script), e.g. via a post-install hook.

**Comment by bogas04** [comment]
npm as well!

**Comment by EivindArvesen** [comment]
[npm](https://www.npmjs.com) is a package manager specifically for JavaScript modules (e.g. for use with [Node.js](https://nodejs.org/en/)), though.

I'm talking about OS package managers, like [Homebrew](http://brew.sh) for Mac OS X, and those that come bundled with various Linux distros (e.g. apt with Debian and Ubuntu).

**Comment by bogas04** [comment]
Yeah it makes sense however having it on npm would make it easy to install on any OS.
I do realize it doesn't really fall under this issue, but I see npm also as a multi-platform package manager. 

Maybe later at some point we can parse the sh files and provide a node module for the same too. 

``` js
require('prm').projects.map((p) => console.log(p));
```

**Comment by EivindArvesen** [comment]
I understand what you mean, but that's not what `npm` is or is for, as far as I understand.

I just don't see a point in distributing prm via a JavaScript package manager when it is not written in JavaScript, and when prm has no place in the JavaScript ecosystem.

On the other hand, if there were a js-binding (and supposably a need for npm's functionality in js/Node), it could be distributed via `npm`, with this repo as a dependency.

**Comment by bogas04** [comment]
Cool, makes sense !

**Comment by bogas04** [comment]
Hey!
I implemented prm using NodeJS coz it looked like a fun exercise! I have mentioned about your project in credits. 

You can have a look at it [here](https://www.npmjs.com/package/node-prm)

![screen shot 2015-09-22 at 8 19 30 pm](https://cloud.githubusercontent.com/assets/6177621/10021777/43ae4d2a-6167-11e5-8ace-5986557ae974.png)

**Comment by EivindArvesen** [comment]
@bogas04: While I appreciate your effort, I would like for you not to use the `prm` name in your project.

This is because, unlike @FredDeschenes' [prm-fish](https://github.com/FredDeschenes/prm-fish), your project is not a port of prm, is not API-compatible, and doesn't perform the same tasks as prm does.

Since your software is very different from prm, this is likely to be very confusing for users.
Please tell me if you disagree with my reasoning, and if so, why.

I'd also appreciate if anyone else would like to chime in here.

**Comment by bogas04** [comment]
Sure, no problem! Off to thinking for name

**Comment by EivindArvesen** [comment]
Great! Good luck with your project:)

**Comment by bogas04** [comment]
Thank you for being so supportive ! :)

**Comment by bogas04** [comment]
[Renamed](https://github.com/bogas04/project-manager-cli) @eivind88  :) `npm i node-pmc -g`

**Comment by EivindArvesen** [comment]
#4 might be relevant.
